### PR TITLE
Pull/mr100

### PR DESCRIPTION
**Pull request by gaionim**

### CODE REVIEW

**Comment by gaionim** [comment]
Add hardware mr100 clone of mini60  clone of sark100

**Comment by zarath** [comment]
Nice, will try to merge it after #212

File: NanoVNASaver/NanoVNASaver.py [hunk 1]
```diff
@@ -609,11 +609,16 @@ def serialButtonClick(self):
...
             self.serialPort = self.serialPortInput.currentData()
+            logger.debug("current data %s", self.serialPortInput.currentData())
+            logger.debug("current text %s", self.serialPortInput.currentText())
             if self.serialPort == "":
                 self.serialPort = self.serialPortInput.currentText()
             logger.info("Opening serial port %s", self.serialPort)
             try:
-                self.serial = serial.Serial(port=self.serialPort, baudrate=115200)
+                if "MR100" in self.serialPortInput.currentText():
+                    self.serial = serial.Serial(port=self.serialPort, baudrate=57600)
```
Comment (zarath) [inline NanoVNASaver/NanoVNASaver.py:619]: Really a buadrate or just set to identify mr100 hardware in the detection part?

Comment (gaionim) [reply NanoVNASaver/NanoVNASaver.py:619]: my mr100 needs 57600, not working on 115200.
So I set wen detected in name and use it to detect when I have only serial.

**Comment by zarath** [comment]
could you add pandas to setup.py and requirements.txt?
I've made a separate branch, as I want to invesitigate if one can circumvent another
library dependency, even if I like pandas..

As sark110 and mr100 are supposed to be similar, perhaps both hardwares can benefit
of code additions.

**Comment by gaionim** [comment]
> could you add pandas to setup.py and requirements.txt?
> I've made a separate branch, as I want to invesitigate if one can circumvent another
> library dependency, even if I like pandas..

deleted requirement,
there is some code for bluetooth connection, but I need refactoring and possibly reuse.
For now is unused  

> As sark110 and mr100 are supposed to be similar, perhaps both hardwares can benefit
> of code additions.

I do not think;
 mr100 is clone of sark100, so minor update needed ( only USBDevice I hope),
but sark110 is better ( and different )

**Pull request merged**